.github/workflows/build.yml

```diff
@@ -131,3 +131,24 @@ jobs:
       with:
         name: Windows Artifacts
         path: ${{ env.INSTALL_NAME }}.zip
+
+  build_windows_msvc:
+    name: Windows (MSVC)
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        arch: [amd64, amd64_x86]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ilammy/msvc-dev-cmd@v1
+        with:
+          arch: ${{ matrix.arch }}
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.x'
+      - name: Install meson and ninja
+        run: pip install meson ninja
+      - name: Configure
+        run: meson setup --buildtype=release --wrap-mode=forcefallback build
```

Comment:
Could you change the `--buildtype` to debug? The idea is that CI builds can be debugged if needed.

Reply:
Ah, now I noticed you didn't added this build to the artifacts, my bad.

Reply:
I uploaded the artifacts with a non-standard `-msvc` and `amd64` and `amd64_x86` suffixes.

Reply:
Such main, only if we can vendor/polyfill the functionality.

Reply:
We should just adopt newer meson versions and tell people to install via pip.